### DISCUSSION TITLE
Resize Album Art

### DISCUSSION THREAD

**Discussion by satanobsidian** [puddletag/puddletag#626] [Q&A] [answered]
Hey guys, 

I can't find a feature to resize the album art within Puddletag like Mp3Tag can do so I've trying to figure out a way but not having much luck. Any advice?

If there is not a built in feature for this, the only other way of doing it I can see is to automate resizing the folder.jpg image through the terminal and somehow replace using Puddletag but can't figure that part out either.

**Comment by ghost** [comment, marked as answer]
I'm not sure if there's a resize function in Puddletag, but you could use [ImageMagick](https://imagemagick.org/script/magick.php) to resize the artwork as long as its not embedded in the audio file.

For instance, to resize `folder.jpg` to 500 pixels wide (preserving aspect ratio):

```sh
magick folder.jpg -resize 500 folder.jpg
```

**Reply by satanobsidian** [reply]
Yeah, I am using ImageMagick to convert then FFmpeg to insert image automatically into audio file. That's the best I could find too. Thanks!
